packages/design-picker/src/components/style.scss

```diff
@@ -388,6 +388,7 @@
 	}
 	.design-picker__grid {
 		@supports ( display: grid ) {
+			row-gap: 40px;
```

Comment:
Currently, the gap between generated designs seems to be `80px`, can we make it `80px` here as well?
![Screen Shot 2022-08-03 at 3 44 06 PM](https://user-images.githubusercontent.com/797888/182552828-f271864a-f1a0-47a7-93d2-8caa0aa60fc5.png)

Reply:
When checking your note I realized the blank space on the left of category menu, should we make them left-aligned too?
<img width="425" alt="image" src="https://user-images.githubusercontent.com/10071857/182574016-e7c5db11-246e-4871-83ac-be090ee8cdc6.png">

Reply:
The design shows the same padding, so it should be fine 👍
![Screen Shot 2022-08-03 at 6 01 36 PM](https://user-images.githubusercontent.com/797888/182581694-68b188df-ebe8-478e-b644-c0747579f697.png)

Reply:
> Currently, the gap between generated designs seems to be 80px, can we make it 80px here as well

@taipeicoder, wait where do you get 80px from? when I look at what's currently in https://horizon.wordpress.com/ it seems to be only 24px?

Reply:
> When checking your note I realized the blank space on the left of category menu, should we make them left-aligned too?

Yes, I think we have to remove the `padding-left` on the mobile to align the figma

![image](https://user-images.githubusercontent.com/13596067/182755903-b6f8684b-b424-4968-93ed-e73d3239484a.png)

Reply:
Okay, it should not be blocker for this, I will create seperate issue and update trigage sheet

Reply:
> > Currently, the gap between generated designs seems to be 80px, can we make it 80px here as well
> 
> @taipeicoder, wait where do you get 80px from? when I look at what's currently in https://horizon.wordpress.com/ it seems to be only 24px?

It's weird, I can also get the value of row-gap 80px yesterday, but now it is 24px again 🤔. Tested on trunk and horizon as well

Reply:
> Currently, the gap between generated designs seems to be 80px, can we make it 80px here as well

Yes, I cannot find it either 🤔

BTW, I think we have to calculate the gap between the premium badge and it's `36px` on the Figma.

![image](https://user-images.githubusercontent.com/13596067/182757002-0dbb6c65-8b2c-4d00-847a-30c0f518c3e0.png)

Also, we have to check with another team (I forgot which team...) why the `design-picker__pricing-description` element uses absolute positioning and it will make the gap looks weird.

Reply:
It looks like they were added here https://github.com/Automattic/wp-calypso/pull/65052 without discussion of absolute vs static position, I imagine it was just what worked at the time. Ok, hmm 36 px to the badge would make it 32px row gap, I think that looks a bit too small:
<img width="573" alt="Screen Shot 2022-08-04 at 2 53 31 pm" src="https://user-images.githubusercontent.com/22446385/182766065-d8bb3713-6d4b-42a9-94c6-3f3ef3b575dc.png">

Reply:
> @taipeicoder, wait where do you get 80px from? when I look at what's currently in https://horizon.wordpress.com/ it seems to be only 24px? 

Just checked again and it's indeed 24px 🤔. Never mind!

Reply:
>  Ok, hmm 36 px to the badge would make it 32px row gap, I think that looks a bit too small:

Yes but I think if we use a static position for the pricing description, it would be ok?

| | |
| - | - |
| ![image](https://user-images.githubusercontent.com/13596067/182774368-1bf527cf-2e87-4696-acb0-6c058d1410d0.png) | ![image](https://user-images.githubusercontent.com/13596067/182774467-eb6b1dfc-8f20-4fe5-8bf4-3d696d2c363c.png) |

Reply:
hmm yea, I think that making the pricing description static positioned is the best solution rather than adding extra padding to compensate for it being absolute. @valterlorran, @markbiek, does that make sense to you two?